src/pulik/atm/Main.java

```diff
@@ -5,31 +5,27 @@
 import pulik.atm.cards.Card;
 import pulik.atm.cards.DebitCard;
 import pulik.atm.machines.TellerMachine;
+import pulik.atm.machines.TerminalATM;
 
 public class Main {
-	public static void main(String[] args) {
+	public static void main(String[] args) throws Exception {
 
-		TellerMachine atm = new TellerMachine();
+		TellerMachine atm = new TerminalATM();
 		Card c = new DebitCard(1234567890123456L, "Peter", "1234", new BigDecimal(199));
 		String inPin = "1234";
 		
 		atm.insert(c);
 		atm.inputPin(inPin);
-	
-		
+		atm.showBalance();
+		atm.eject();
 		
-		try { 
-			atm.showBalance();
-		}
+		System.out.println("\n\n");
+	
+		atm.insert(c);
+		atm.showBalance();
+		atm.eject();
```

Comment:
These rows are not really necessary. This was just to demonstrate the `showBalance()` won't work without calling `insertPin()` with correct PIN. We can keep it there for now though...